frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -110,6 +111,8 @@ export default memo(function Filters({
     ethnicities,
     sexes,
   } = selectedFilters;
+  const { pathname } = useRouter();
+  const isVersion2 = pathname.includes("v2");
```

Comment:
maybe worth extracting this as a hook when we use this in 3 or more places? Right now it's only twice, so probably not worth extracting

Reply:
Sounds good, will probably help with the inevitable merge into main gene-expression